pkg/transferer/transferer.go

```diff
@@ -32,9 +32,12 @@ type Transferer struct {
 	sources      []dump.Source
 	workersCount int
 	file         io.ReadWriter
+	encrypted    *bool
+	key          *string
+	iv           *string
```

Comment:
I think it's better to move this into a separate struct for clearness. For example, we could name it `EncryptionOptions`